components/file_upload/file_upload.test.tsx

```diff
@@ -208,18 +209,19 @@ describe('components/FileUpload', () => {
         event.preventDefault = jest.fn();
         const getAsFile = jest.fn().mockReturnValue(new File(['test'], 'test'));
         const file = {getAsFile, kind: 'file', name: 'test'};
-        event.clipboardData = {items: [file], types: ['image/png']};
+        (event as any).clipboardData = {items: [file], types: ['image/png']};
 
         const wrapper = shallowWithIntl(
             <FileUpload
                 {...baseProps}
             />,
         );
-        jest.spyOn(wrapper.instance(), 'containsEventTarget').mockReturnValue(true);
-        const spy = jest.spyOn(wrapper.instance(), 'checkPluginHooksAndUploadFiles');
+        const instance = wrapper.instance() as FileUploadClass;
+        jest.spyOn(instance, 'containsEventTarget').mockReturnValue(true);
+        const spy = jest.spyOn(instance, 'checkPluginHooksAndUploadFiles');
 
         if (!fileSupported) {
-            global.File = undefined;
+            global.File = {} as typeof RealFile;
```

Comment:
Tests are failing because of this. I have a dilemma here 😅 

The test is testing `should upload file on paste when File constructor is not supported` and test is failing because `Uncaught [TypeError: File is not a constructor]`

Any idea how to fix this?

Reply:
Do we even need this part of the test any more? As far as I can tell, `fileSupported` should be true on every non-IE11 browser

Reply:
Yes, you are right. Anyhow, this part was removed by Michel as part of another PR